src/components/collection/CollectionForm.js

```diff
@@ -97,6 +97,12 @@ const schema = {
       title: "Sort",
       default: "-last_modified",
     },
+    cache_expires: {
+      type: "integer",
+      title: "Cache expires",
+      default: 0,
+      description: "(in seconds) add client cache headers on read-only requests."
```

Comment:
Could you please link to [this section](http://kinto.readthedocs.io/en/stable/api/1.x/collections.html#collection-caching) of the kinto docs so users know about what this does precisely?

Hint: you can use HTML there:

```jsx
description: <p>(in seconds) add <a href="">client cache headers on read-only requests</a>.</p>
```